x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/rule_migrations_task_client.ts

```diff
@@ -169,7 +169,7 @@ export class RuleMigrationsTaskClient {
         this.logger.info(`Abort signal received, stopping migration ID:${migrationId}`);
         return;
       } else {
-        this.logger.error(`Error processing migration ID:${migrationId}`, error);
+        this.logger.error(`Error processing migration ID:${migrationId} ${error}`, error);
```

Comment:
A bit unsure what is common here, because there is no error displayed unless I did this? What is the second argument supposed to do in this context?

Reply:
You are right, the second argument is the `meta` object, this is wrong. Let's apply your change, and can you remove the second parameter as well?